include/envoy/server/drain_manager.h

```diff
@@ -21,6 +21,8 @@ class DrainManager : public Network::DrainDecision {
    */
   virtual void startDrainSequence(std::function<void()> drain_complete_cb) PURE;
 
+  virtual bool draining() const PURE;
```

Comment:
Add doc comments for the interface method.

Reply:
Done, thanks for the quick turnaround.